exercises/word-search/canonical-data.json

```diff
@@ -49,14 +197,140 @@
         "wolcqlirpc",
         "screeaumgr",
         "alxhpburyi",
+        "clojurermt",
+        "jalaycalmp"
+      ],
+      "wordsToSearchFor": [
+        "clojure"
+      ],
+      "expected": {
+        "clojure": {
+          "start": {
+            "column": 1,
+            "row": 9
+          },
+          "end": {
+            "column": 7,
+            "row": 9
+          }
+        }
+      }
+    },
+    {
+      "description": "Should locate a different left to right word in a ten line grid",
+      "property": "search",
+      "grid": [
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "fortranftw",
+        "alxhpburyi",
+        "clojurermt",
+        "jalaycalmp"
+      ],
+      "wordsToSearchFor": [
+        "fortran"
+      ],
+      "expected": {
+        "fortran": {
+          "start": {
+            "column": 1,
+            "row": 7
+          },
+          "end": {
+            "column": 7,
+            "row": 7
+          }
+        }
+      }
+    },
+    {
+      "description": "Should locate multiple words",
+      "property": "search",
+      "grid": [
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "fortranftw",
```

Comment:
> fortran FTW

subtle inducement to become a fortran fan

Reply:
Yep.  Dating myself...